packages/office-addin-debugging/src/commands.ts

```diff
@@ -23,14 +38,13 @@ function parseDevServerPort(optionValue: any): number | undefined {
     return devServerPort;
 }
 
-export async function start(manifestPath: string, appType: string | undefined, command: commander.Command) {
+export async function start(manifestPath: string, platform: string | undefined, command: commander.Command) {
     try {
-        const appTypeToDebug: AppType | undefined = parseAppType(appType || process.env.npm_package_config_app_type_to_debug || "desktop");
+        const appPlatformToDebug: AppPlatform | undefined = parseAppPlatform(platform || process.env.npm_package_config_app_platform_to_debug || AppPlatform.Win32);
```

Comment:
Please rename to `platformToDebug`. 
Please use `npm_package_config_platform_to_debug`,